apps/asap-server/src/migrations/1640102001540-set-ros-added-date-default.ts

```diff
@@ -0,0 +1,35 @@
+/* istanbul ignore file */
+import { RestResearchOutput, Results, SquidexRest } from '@asap-hub/squidex';
+import { Migration } from '../handlers/webhooks/webhook-run-migrations';
+
+export default class SetResearchOutputAddedDateDefault extends Migration {
+  up = async (): Promise<void> => {
+    const squidexClient = new SquidexRest<RestResearchOutput>(
+      'research-outputs',
+    );
+
+    let pointer = 0;
+    let result: Results<RestResearchOutput>;
+
+    do {
```

Comment:
there is a helper function that does this for you, see: https://github.com/yldio/asap-hub/blob/98c32d3efe7b615b8466f4f6f093727b54082741/apps/asap-server/src/utils/migrations.ts#L4

you may wanna consider using that as it has been used before. We've had issues with other attempts at looping through collections which is why I think its quite useful to stick to that :D